tensorflow/cc/framework/testutil.h

```diff
@@ -26,9 +26,21 @@ namespace test {
 void GetTensors(const Scope& scope, OutputList tensors,
                 std::vector<Tensor>* out);
 
+// Computes the outputs listed in 'tensors', returns the tensors in 'out'.
+// Take a vector of Output as extra parameters if some Variable should
+// be assigned before the session run call.
```

Comment:
Grammar nit: change the second sentence to: 
`assign_vars` are extra outputs that should be run e.g. to assign values to variables.